manifest.go

```diff
@@ -171,10 +196,11 @@ func readManifest(r io.Reader) (*Manifest, []error, error) {
 
 func fromRawManifest(raw rawManifest) (*Manifest, error) {
 	m := &Manifest{
-		Constraints: make(gps.ProjectConstraints, len(raw.Constraints)),
-		Ovr:         make(gps.ProjectConstraints, len(raw.Overrides)),
-		Ignored:     raw.Ignored,
-		Required:    raw.Required,
+		Constraints:  make(gps.ProjectConstraints, len(raw.Constraints)),
+		Ovr:          make(gps.ProjectConstraints, len(raw.Overrides)),
+		Ignored:      raw.Ignored,
+		Required:     raw.Required,
+		PruneOptions: gps.PruneNestedVendorDirs,
```

Comment:
Seems like it's not just a default but a required "base flag" that is always set and manifest cannot unset this flag?